__tests__/serializers/combined/fixtures/elementTags.ts

```diff
@@ -1,6 +1,7 @@
 interface Ifixture {
   name: string
   html: string
+  htmlFromSlate?: string
```

Comment:
Nice addition, and makes sense. I remember combined fixtures were the first tests I wrote, but now that the serializers have developed further, it's become clear that we can't expect the same result both ways.